x-pack/platform/plugins/shared/lens/server/content_management/v1/cm_services.ts

```diff
@@ -54,7 +54,7 @@ const lensSavedObjectSchema = schema.object(
   { unknowns: 'allow' }
 );
 
-const getResultSchema = schema.object(
+export const lensGetResultSchema = schema.object(
   {
     item: lensSavedObjectSchema,
     meta: schema.object(
```

Comment:
Do we want to return all this other `meta` data from the API? Typical with get APIs, I'd expect just the value, not a nested object value (i.e. like [GitHub's get issue](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#get-an-issue)).